Makefile

```diff
@@ -315,6 +316,12 @@ test-xxhsum-c: xxhsum
 	echo "00000000  test-expects-file-not-found" | ./xxhsum -c -; test $$? -eq 1
 	@$(RM) .test.*
 
+LIB_FUZZING_ENGINE?="-fsanitize=fuzzer"
+ifeq '' '$(findstring clang++,$(CXX))'
```

Comment:
Is this `ifeq` correct ?
I thought it would require parenthesis

Reply:
Your probably right. Just fiddling with this to get it working.